lib/lambda/processEmails.ts

```diff
@@ -123,8 +125,12 @@ export async function processRecord(kafkaRecord: KafkaRecord, config: ProcessEma
     console.log("Kafka event is not of mako origin.  Doing nothing.");
     return;
   }
-
-  await processAndSendEmails(record, id, config);
+  try {
+    await processAndSendEmails(record, id, config);
+  } catch (error) {
+    console.error("Error processing record:", JSON.stringify(error, null, 2));
+    throw new Error(`Error occured while attempting to process and send emails`);
```

Comment:
Should we include the record in the thrown Error? Like you did in the `console.error` on the line before